keyboards/massdrop/ctrl/keymaps/xanimos/keymap.c

```diff
@@ -0,0 +1,394 @@
+#include "keymap.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15


Also, `keymap.h` is used internally by QMK Firmware.  It may be best to use a different name here.

Reply:
Resolved copyright in cca5af2
As for the keymap.h, I was following how the other keymaps have in inside the massdrop/ctrl as well as the standard keep the same name as the .c file. the keymap.h has an include with the global defined QMK_KEYBOARD_H